libraries/api-extractor/src/DocElementParser.ts

```diff
@@ -134,30 +86,27 @@ export default class DocElementParser {
 
         for (const paragraph of token.text.split(/\n\s*\n/g)) {
           if (!firstLoop) {
-            docElements.push({
-              kind: 'paragraphDocElement'
-            } as IParagraphElement);
+            markupElements.push(Markup.PARAGRAPH);
           }
           firstLoop = false;
 
           const normalizedParagraph: string = paragraph.replace(/\s+/g, ' ').trim();
           if (normalizedParagraph) {
-            docElements.push({
-              kind: 'textDocElement',
-              value: ' ' + normalizedParagraph + ' '
-            } as ITextElement);
+            markupElements.push(
+              ...Markup.createTextElements(' ' + normalizedParagraph + ' ')
```

Comment:
>' ' [](start = 43, length = 3)

are you sure we need this?

Reply:
Good point.  The problem this was trying to solve is already handled by getTrimmedSpan().  I moved that function to be Markup.normalize() and applied it everywhere.

---
In reply to: [146729341](https://github.com/Microsoft/web-build-tools/pull/411#discussion_r146729341) [](ancestors = 146729341)